packages/core/src/domain/configuration/transportConfiguration.ts

```diff
@@ -72,15 +72,7 @@ function computeReplicaConfiguration(
   return assign({ applicationId: initConfiguration.replica.applicationId }, replicaEndpointBuilders)
 }
 
-function computeIntakeUrlPrefixes(
-  endpointBuilders: ReturnType<typeof computeEndpointBuilders>,
-  site: string
-): string[] {
-  const intakeUrlPrefixes = objectValues(endpointBuilders).map((builder) => builder.urlPrefix)
-
-  if (site === INTAKE_SITE_US1) {
-    intakeUrlPrefixes.push(`https://${PCI_INTAKE_HOST_US1}/`)
-  }
-
-  return intakeUrlPrefixes
+function hasIntakeParameters(url: string) {
+  // check if tags is present in the query string
+  return url.indexOf('ddsource') !== -1 || url.indexOf('ddtags') !== -1
```

Comment:
**💬 suggestion:** Be a bit stricter? URLs should have both params, you could use `&&` instead of `||`

**🥜 nitpick:** ‏use `includes()` (from polyfills.ts) instead of `indexOf`. It's a bit annoying because of v6 though...